test/test_browser.py

```diff
@@ -5524,6 +5525,14 @@ def test_error_reporting(self):
     create_file('post.js', 'throw "foo";')
     self.btest(test_file('hello_world.c'), args=['--post-js=post.js'], expected='exception:foo')
 
+  def test_webpack(self):
+    shutil.copytree(test_file('webpack'), 'webpack')
+    with utils.chdir('webpack'):
+      self.compile_btest([test_file('hello_world.c'), '-sEXIT_RUNTIME', '-sMODULARIZE', '-sENVIRONMENT=web', '-o', 'src/hello.js'])
```

Comment:
Might want to have a separate one for `EXPORT_ES6` too as it intentionally emits some bundler-specific tricks since https://github.com/emscripten-core/emscripten/pull/14135.

Reply:
Yes, I'm planning on adding more tests of different bundlers and using ES6 modules.

BTW, I was thinking of make `-sMODULARIZE` take a string.  So you would do `-sMODULARIZE=ES6` or `-sMODULARIZE=CJS` or `-sMODULARIZE=AMD` ..  then we could think about perhaps one day changing the default `-sMODULARIZE` from CJS to ES6 at some point.

Reply:
*shrug* I'm not sure it's worth investing in other old formats like AMD to the point where we'd have a separate config for them, and EXPORT_ES6 already works on its own w/o explicit MODULARIZE.

Reply:
Oh cool, I forgot about that. 

I'll add some more bundler tests with ES6 then

Reply:
I can try my hand at these as I have already put a few together here: https://github.com/dhdaines/webpack-wasm-test